public/app/features/connections/tabs/ConnectData/CardGrid/CardGrid.tsx

```diff
@@ -0,0 +1,74 @@
+import { css } from '@emotion/css';
+import React, { FC } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { Card, LoadingPlaceholder, useStyles2 } from '@grafana/ui';
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  sourcesList: css`
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(280px, 1fr));
+    gap: 12px;
```

Comment:
Nit: maybe we should the `theme.spacing(1.5)` here? Although as we would need to use "1.5" maybe it doesn't make sense.